ghga_transpiler/config_loader.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Module to load yaml file and return the config onject"""
+
+from importlib import resources
+
+import yaml
+
+from ghga_transpiler.config.config import Config
```

Comment:
The two equally named submodules are confusing IMHO. I would suggest to just have one `config.py` file / module under `ghga_transpiler` and another `config` module (as a folder) - not nested, but independent both under `ghga_transpiler`.